ui-tests/cypress/integration/administration/email-configuration-Sender.spec.js

```diff
@@ -0,0 +1,306 @@
+
+
+      });
+      it([Tag.SMOKE, 'ENG-3299'], 'Edit Sender is displayed', () => {
+
+
+        currentPage.getContent().getActionButton().click();
+        currentPage.getContent().getEditButton().click().wait(1000);
+        cy.validateAppBuilderUrlPathname('/email-config/senders/edit/CODE1');
+        currentPage.getContent().getSenderForm()
+                   .should('be.visible')
+                   .and('contain', 'Code ')
+                   .and('contain', 'Email ');
+        currentPage.getContent()
+                   .getCodeInput()
+                   .should('be.visible')
+                   .and('have.value', defaultCode1);
+        currentPage.getContent()
+                   .getEmailInput()
+                   .should('be.visible')
+                   .and('have.value', defaultEmail1);
+
+
+      });
+
+
+
+
+    });
+
+    describe([Tag.SANITY, 'ENG-3299'], 'Click on Buttons', () => {
+
+
+      it([Tag.SANITY, 'ENG-3299'], 'Add a new Sender', () => {
+
+
+        currentPage.getContent().getAddButton().click().wait(1000);
+        currentPage.getContent()
+                   .getCodeInput()
+                   .type(testCode);
+        currentPage.getContent()
+                   .getEmailInput()
+                   .clear()
+                   .type(testEmail);
+        currentPage.getContent().senderSubmit()
+                   .click().wait(500);
+        currentPage.getContent()
+                   .getSenderTable()
+                   .contains(testEmail)
+                   .should('be.visible');
+        cy.validateToast(currentPage);
+
+       cy.senderController()
+         .then(controller => controller.deleteSender(testCode)).wait(1000);
+
+      });
+
+      it([Tag.SANITY, 'ENG-3299'], 'Update an existing sender', () => {
+
+        currentPage.getContent().getActionButton().click();
+        currentPage.getContent().getEditButton().click().wait(1000);
+        cy.validateAppBuilderUrlPathname('/email-config/senders/edit/CODE1');
+        currentPage.getContent()
+                   .getEmailInput()
+                   .clear()
+                   .type(testEmail);
+        currentPage.getContent().senderSubmit()
+                   .click().wait(500);
+        currentPage.getContent()
+                   .getSenderTable()
+                   .contains(testEmail)
+                   .should('be.visible');
+        cy.validateToast(currentPage);
+        cy.senderController()
+          .then(controller => controller.defaultEditSender(defaultCode1)).wait(1000);
+
+      });
+
+      it([Tag.SANITY, 'ENG-3299'], 'Delete Modal is displayed', () => {
+
+
+        currentPage.getContent().getActionButton().click();
+        currentPage.getContent().getDeleteButton().click().wait(1000);
+        currentPage.getDialog()
+                   .get().children(`${htmlElements.div}#DeleteSenderModal`)
+                   .should('be.visible');
+
+       });
+
+      it([Tag.SANITY, 'ENG-3299'], 'Delete a sender', () => {
```

Comment:
Please, do not delete a default value, other tests might rely on it; create a new one via API and then delete it via UI instead